script.js

```diff
@@ -1,73 +1,67 @@
-let books = [];
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
-}
-
-const deleteBook = (title, author) => {
-  books = books.filter((book) => book.title !== title || book.author !== author);
-};
-
-const setLocalStorage = (books) => {
-  const data = JSON.stringify(books);
-  localStorage.setItem('books', data);
-};
-
 const bookContainer = document.querySelector('.book-container');
+const inpTitle = document.querySelector('#title');
+const inpAutor = document.querySelector('#author');
+class Book {
+    static collection = [];
 
-function showBook(book) {
-  const bookDiv = document.createElement('DIV');
-  const titleBook = document.createElement('P');
-  const authorBook = document.createElement('P');
-  const deleteBtn = document.createElement('BUTTON');
-  const hr = document.createElement('hr');
-  deleteBtn.textContent = 'Remove';
+    constructor(title, author) {
+      this.title = title;
+      this.author = author;
+    }
 
-  titleBook.textContent = book.title;
-  authorBook.textContent = book.author;
+    static setLocalStorage=() => {
+      const data = JSON.stringify(Book.collection);
+      localStorage.setItem('books', data);
+    }
 
-  bookDiv.appendChild(titleBook);
-  bookDiv.appendChild(authorBook);
-  bookDiv.appendChild(deleteBtn);
-  bookDiv.appendChild(hr);
+    deleteBook = (title, author) => {
+      Book.collection = Book.collection.filter((book) => book.title !== title
+      || book.author !== author);
+    }
```

Comment:
- [ ] Good job with refactoring the code 👍. However, I added 3 books with the same title and author and when I removed one and refresh the page, the remaining two books were removed. To make sure this is resolved properly, consider using a unique id property for each book since it would make it easier to remove books with duplicate values 😄.